scene/gui/split_container.cpp

```diff
@@ -258,7 +258,7 @@ Size2 SplitContainer::get_minimum_size() const {
 	int sep = (dragger_visibility != DRAGGER_HIDDEN_COLLAPSED) ? MAX(theme_cache.separation, vertical ? g->get_height() : g->get_width()) : 0;
 
 	for (int i = 0; i < 2; i++) {
-		if (!_get_sortable_child(i)) {
+		if (!_get_sortable_child(i, SortableVisbilityMode::VISIBLE)) {
```

Comment:
It would be nice if this Control was assigned to a variable instead of calling the method again below.